src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public static final Random random = new Random();
```

Comment:
The `random` instance should not be `public` or `static`. It's better to make it private and non-static to avoid sharing it across multiple instances of `ColorSupplier`.